xarray/core/computation.py

```diff
@@ -579,28 +589,41 @@ def apply_variable_ufunc(
                 "``.load()`` or ``.compute()``"
             )
         elif dask == "parallelized":
-            input_dims = [broadcast_dims + dims for dims in signature.input_core_dims]
             numpy_func = func
 
+            if dask_gufunc_kwargs is None:
+                dask_gufunc_kwargs = {}
+
             def func(*arrays):
-                return _apply_blockwise(
-                    numpy_func,
-                    arrays,
-                    input_dims,
-                    output_dims,
-                    signature,
-                    output_dtypes,
-                    output_sizes,
-                    meta,
+                import dask.array as da
+
+                res = da.apply_gufunc(
+                    numpy_func, str(signature), *arrays, **dask_gufunc_kwargs,
                 )
 
+                # todo: covers for https://github.com/dask/dask/pull/6207
+                #  remove when minimal dask version >= 2.17.0
+                if signature.num_outputs > 1:
+                    res = (*res,)
```

Comment:
Does this work any differently from `tuple(res)`? If not, then I would slightly lean towards using `tuple()` explicitly. This newer syntax is a little unfamiliar to me, but maybe I'll get used to it eventually :)

Reply:
According to some resources at [SO](https://stackoverflow.com/questions/30216000/why-is-faster-than-list) use of literal syntax is faster than using  `tuple()`. I've no strong opinion about that and can change it to `tuple()`.

Reply:
changed to tuple, adapt dask version check